README.md

```diff
@@ -136,6 +143,10 @@ Alternatively disable signing in each target:
 * Select `Add User-Defined Setting`
 * Add `CODE_SIGNING_REQUIRED` setting with a value of `NO`
 
+### Swift Package Manager
+* `open Package.swift`
+* Xcode will open, then choose a scheme for a library to build or test suite to run.
```

Comment:
Nit: should we mention that you can develop for Catalyst, macOS, tvOS, watchOS by selecting the run destination?